apps/server/src/shared/common/interceptor/duration-logging.interceptor.ts

```diff
@@ -13,8 +13,8 @@ export class DurationLoggingInterceptor implements NestInterceptor {
 	}
 
 	intercept(context: ExecutionContext, next: CallHandler): Observable<unknown> {
-		this.logger.verbose('Before...');
+		this.logger.log('Before...');
```

Comment:
I create a ticket to use the new logger in any of the core/shared parts.

Reply:
This interceptor is not in use, maybe we cal remove it